File: Java/Lab1/src/Main.java
import Guitar.Guitar;

public class Main {
    public static void main(String[] args) {
        // Create a new Guitar object
        Guitar myGuitar = new Guitar("Fender", "Stratocaster", "Sunburst");

        // Access and print attributes
        System.out.println("Brand: " + myGuitar.getBrand());
        System.out.println("Model: " + myGuitar.getModel());
        System.out.println("Color: " + myGuitar.getColor());

        // Call methods
        myGuitar.playChord("G major");
        myGuitar.tune();

        // Update attributes using the setters
        myGuitar.setBrand("Gibson");
        myGuitar.setColor("Cherry Sunburst");

        // Print updated attributes
        System.out.println("\nUpdated Attributes:");
        System.out.println("Brand: " + myGuitar.getBrand());
        System.out.println("Model: " + myGuitar.getModel());
        System.out.println("Color: " + myGuitar.getColor());
    }
}

File: Python/Lab1/main.py


class Guitar:
    def __init__(self, brand, model, color):
        self.brand = brand
        self.model = model
        self.color = color

    def playChord(self, chord):
        print(f"Playing the chord: {chord}")

    def tune(self):
        print("Tuning the guitar.")

    def setBrand(self, brand):
        self.brand = brand

    def setModel(self, model):
        self.model = model

    def setColor(self, color):
        self.color = color

    def getBrand(self):
        return self.brand

    def getModel(self):
        return self.model

    def getColor(self):
        return self.color

# Create a new Guitar object
my_guitar = Guitar("Fender", "Stratocaster", "Sunburst")

# Access and print attributes
print("Brand:", my_guitar.getBrand())
print("Model:", my_guitar.getModel())
print("Color:", my_guitar.getColor())

# Call methods
my_guitar.playChord("G major")
my_guitar.tune()

# Update attributes using the setters
my_guitar.setBrand("Gibson")
my_guitar.setColor("Cherry Sunburst")

# Print updated attributes
print("\nUpdated Attributes:")
print("Brand:", my_guitar.getBrand())
print("Model:", my_guitar.getModel())
print("Color:", my_guitar.getColor())